deploy/helm/templates/addons/minio.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   type: Helm
 
   helm:
-    chartLocationURL: oci://registry-1.docker.io/bitnamicharts/minio
+    chartLocationURL: https://jihulab.com/api/v4/projects/152630/packages/helm/stable/charts/minio-12.8.12.tgz
 
     installValues:
```

Comment:
What is the docker image url, can it use own image.

Reply:
I direct change the minio's chart. All docker images are own image in aliyun now.